### ISSUE TITLE
Can't queue/play Spotify playlist 

### ISSUE THREAD

**Issue by kaaholst** [kaaholst/android-squeezer#80]
```
What steps will reproduce the problem?
1. Go to Spotify playlists.
2. There's no options available to queue/play the playlist. 
3. If you click/enter playlist, you have option for every track to play/queue. 

Screen shot provided showing playlists with no option available. ie no little arrow
next to playlist. 

What is the expected output? What do you see instead?

- Option available to play/queue/etc Spotify playlists 


What version of the product are you using? On what operating system?

- Dec 22, 2013
- Android 4.3
Please provide any additional information below.

```

Original issue reported on code.google.com by `jerano` on 2014-01-08 18:28:37

<hr>
- _Attachment: Screenshot_2014-01-08-19-27-04.png<br>![Screenshot_2014-01-08-19-27-04.png](https://storage.googleapis.com/google-code-attachments/android-squeezer/issue-54/comment-0/Screenshot_2014-01-08-19-27-04.png)_

**Comment by kaaholst** [comment]
```
Please, not even a simple comment regarding this from the devs, or am I the only one
using the Spotify-plugin on the server? Kind of a major bug not being able to queue
playlists or even start playing a playlist in Spotify. I can't believe it's a _major_
hassle to implement?
```

Original issue reported on code.google.com by `jerano` on 2014-12-13 12:17:05

**Comment by kaaholst** [comment]
```
I like to help you with this issue.

To do that I need some assistance. To start with please post the step you perform to
"Go to Spotify playlists", preferably accompanied by screenshots.

Also a screenshot of your server settings, plugins tab - showing the Spotify plugin
please.


```

Original issue reported on code.google.com by `kaaholst` on 2014-12-23 11:20:54

**Comment by kaaholst** [comment]
```
Hello and thanks!

At Squeezer main menu / See numbered screenshots!
1 - Select "Internet Radio"
2 - Select "Spotify"
3 - Select "Playlists" (Here I think every playlist should have a submenu, right?!)
4 - Select a spotify playlist.
5 - Now you have a list of songs in this playlist.
6 - For each song in the playlist, you have submenu for Play/Queue/Play Next

I think what is really missing, is the same little menu for each playlist,
that when pressed gives you options like Play/Queue/Play Next.
Like the menu you can get for each song in the playlist - See Screenshot 5 and 6 !


Thanks in advance! Keep up the good work! This app is what's keeping me happy with
my Squeezebox!!

Ps. Maybe "Play", "Queue" and "Play Next" are easier/faster to read and understand
than
"Play", "Add to playlist" and "Play after the currently playing song"? Just a thought....
Ds.


```

Original issue reported on code.google.com by `jerano` on 2014-12-25 13:05:29

<hr>
- _Attachment: 1-select_internet_radio.png<br>![1-select_internet_radio.png](https://storage.googleapis.com/google-code-attachments/android-squeezer/issue-54/comment-3/1-select_internet_radio.png)_
- _Attachment: 2-select_spotify.png<br>![2-select_spotify.png](https://storage.googleapis.com/google-code-attachments/android-squeezer/issue-54/comment-3/2-select_spotify.png)_
- _Attachment: 3-select_playlists.png<br>![3-select_playlists.png](https://storage.googleapis.com/google-code-attachments/android-squeezer/issue-54/comment-3/3-select_playlists.png)_
- _Attachment: 4-select_a_playlist.png<br>![4-select_a_playlist.png](https://storage.googleapis.com/google-code-attachments/android-squeezer/issue-54/comment-3/4-select_a_playlist.png)_
- _Attachment: 5-spotify_playlist-list.png<br>![5-spotify_playlist-list.png](https://storage.googleapis.com/google-code-attachments/android-squeezer/issue-54/comment-3/5-spotify_playlist-list.png)_
- _Attachment: 6-spotify_playlist_object_action.png<br>![6-spotify_playlist_object_action.png](https://storage.googleapis.com/google-code-attachments/android-squeezer/issue-54/comment-3/6-spotify_playlist_object_action.png)_
- _Attachment: Server-Settings.png<br>![Server-Settings.png](https://storage.googleapis.com/google-code-attachments/android-squeezer/issue-54/comment-3/Server-Settings.png)_
- _Attachment: Server-Settings-Advanced-Plugin.png<br>![Server-Settings-Advanced-Plugin.png](https://storage.googleapis.com/google-code-attachments/android-squeezer/issue-54/comment-3/Server-Settings-Advanced-Plugin.png)_

**Comment by kaaholst** [comment]
```
Thanks, I'll see what I can do

About the wording, "Play", "Queue" and "Play Next", my initial thinking, is that you
are right. I feel however, that the wording should be consistent throughout the app,
so if it's changed, it should be the same for all playable items. I suggest you open
a discussion on the group: Squeezer <android-squeezer@googlegroups.com>, or create
a separate issue.

```

Original issue reported on code.google.com by `kaaholst` on 2014-12-28 18:39:01

**Comment by kaaholst** [comment]
Fixed in 2.0.0

Original issue reported on github.com/nikclayton/android-squeezer by kaaholst on 2020-03-07T19:10:48Z
